products/batch_exports/backend/tests/temporal/utils/s3.py

```diff
@@ -55,3 +55,13 @@ def read_s3_data_as_json(data: bytes, compression: str | None) -> list:
 
     json_data = [json.loads(line) for line in data.decode("utf-8").split("\n") if line]
     return json_data
+
+
+async def delete_all_from_s3(s3_client, bucket_name: str, key_prefix: str):
```

Comment:
Moved to common test utilities module as we now use this when testing redshift too.